### DISCUSSION TITLE
Registration, token not valid

### DISCUSSION THREAD

**Discussion by ronzle** [solectrus/solectrus#3603] [General]
Hi 

how can fix this invalid token issue, to participate the sponsoring ? 

[screenshot deleted]

Thanks
Ronny

**Comment by ledermann** [comment]
Wegen des deutschen Screenshots antworte ich der Einfachheit halber auf Deutsch.

Wenn ein zahlungswilliger Nutzer wegen einer Fehlermeldung nicht weiterkommt, ist das natürlich nicht so gut ;-) 

Wegen der nicht eindeutig lesbaren URL im Screenshot kann ich das nicht gut untersuchen. Ich habe noch keinerlei Idee, woran das liegen könnte. Könntest du mir die URL (oder eine andere, die den gleichen Fehler verursacht), **per E-Mail** an info@solectrus.de schicken?

Grundsätzlich läuft es so: In SOLECTRUS selbst (also der eigenen Instanz) gelangt man über den Link "Registrierung" in die Kontoverwaltung auf `registration.solectrus.de`. Von da aus kann man dann das Sponsoring-Abo klicken.

Du kommst also überhaupt nicht mehr in die Registrierung? Welcher Browser? Kannst du einen anderen Browser ausprobieren? Hilft es, die Cookies zu löschen? 

Grundsätzlich bitte immer auf den Button "Registrierung" in SOLECTRUS klicken, nicht den Link woanders herkopieren oder den gleichen Link mehrfach versuchen, der Link ist nur einmal gültig.

**Comment by ledermann** [comment]
Geklärt per E-Mail
